doc/architecture/database_schemata.md

```diff
@@ -146,6 +146,41 @@ maps }o--|| users : "owned by"
 blossoms ||--o{ blossoms_gained : ""
 blossoms_gained }o--|| users : ""
 
+ingredientLists {
+  INT id PK
+  VARCHAR name "NOT NULL"
+  VARCHAR description
+  BYTEA image
+  BOOLEAN is_recurring "NOT NULL"
+  DATE end_date "NOT NULL"
+  INT accomplished
+}
+
+ingredients {
+  INT needed "NOT NULL"
```

Comment:
Let us keep it more simple: without any amount the user simply states if enough could be harvested.